Dockerfile

```diff
@@ -15,5 +15,11 @@ COPY . /partmc/
 
 RUN mkdir /build \
     && cd /build \
-    && cmake /partmc \
+    && cmake -D CMAKE_BUILD_TYPE=release \
+             -D CMAKE_C_FLAGS_DEBUG="-g" \
+             -D CMAKE_Fortran_FLAGS_DEBUG="-g" \
+             -D CMAKE_C_FLAGS_RELEASE="-O2 -g -Werror -Wall -Wextra" \
+             -D CMAKE_Fortran_FLAGS_RELEASE="-O2 -g -Werror -fimplicit-none -Wall -Wextra -Wconversion -Wunderflow -Wimplicit-interface -Wno-compare-reals -Wno-unused -Wno-unused-parameter -Wno-unused-dummy-argument -fbounds-check" \
+             -D ENABLE_GSL:BOOL=TRUE \
```

Comment:
@jcurtis2 can we also turn on sundials and other standard options here?

Reply:
We have to address #33 for our condensation code to work with newer versions of SUNDIALS.

Reply:
https://github.com/compdyn/partmc/pull/167